sippy-ng/src/component_readiness/CompReadyTestReport.js

```diff
@@ -251,30 +251,41 @@ export default function CompReadyTestReport(props) {
       return itemB - itemA
     })
 
+    if (!versions[version]) {
+      // Handle the case where GA date is undefined (implies under development and not GA)
+      const weeksBefore = Math.floor(
+        (toDate - fromDate) / (1000 * 60 * 60 * 24 * 7)
+      )
+      return `Recent ${weeksBefore} week(s) of ${version}`
+    }
```

Comment:
Imagine I set the time span to 5/1/2021 to 5/31/2021 - it'd report "Recent 5 weeks" even though it's 2 years ago

Reply:
yikes, let me look into that.